tests/signer/test_signer.py

```diff
@@ -108,24 +104,21 @@ def test_signer_get_policy_hash(signer):
         b"g7zRG27piJkJOpflGu7XACpMj9hC6dVMcCyzqHxxPZeQ==")
 
 
-def test_signer_prepare_issuer_name(signer, pkcs12_certificate):
-    certificate, password = pkcs12_certificate
-    certificate_object = signer._parse_certificate(certificate, password)
+def test_signer_prepare_issuer_name(signer, certificate):
+    certificate_object = signer._parse_certificate(certificate)
     certificate = certificate_object.get_certificate()
     issuer_name = signer._prepare_issuer_name(certificate)
     assert issuer_name == (
-        b'emailAddress=info@mit-xperts.com,CN=itv.mit-xperts.com,'
+        b'1.2.840.113549.1.9.1=info@mit-xperts.com,CN=itv.mit-xperts.com,'
```

Comment:
This OID seems to be the correct representation of an email adress, this is what is also used by the dian exemplification:
```xml
<ds:X509IssuerName>
C=CO,L=Bogota D.C.,O=Andes SCD.,OU=Division de certificacion entidad final,CN=CA ANDES SCD S.A. Clase II,1.2.840.113549.1.9.1=#1614696e666f40616e6465737363642e636f6d2e636f
</ds:X509IssuerName>
```